challenge.js

```diff
@@ -58,4 +67,5 @@
 
 // ANSWER
 
+//http://www.w3schools.com/jsref/met_node_appendchild.asp
```

Comment:
Use MDN, not w3schools for JS! MDN is *the* highest quality source for JS documentation at the present moment.

The method you chose is correct. Since you mentioned `document.createElement` above, here is a snippet using the both of them:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1